### PR TITLE
Ejs loading

### PR DESCRIPTION
**Pull request by CLIcoder**

### CODE REVIEW

File: engine/components/network/net.io/net.io-client/index.js [hunk 1]
```diff
@@ -295,16 +295,19 @@ NetIo.Client = NetIo.EventingClass.extend({
 		// 	}
 		// }
 
+		// KILL SERVER MEANS THAT THE GAME WAS UNPUBLISHED ON STATE 3 AND STATE 2
```
Comment (sloont) [inline engine/components/network/net.io/net.io-client/index.js:298]: seems like we could add two different messages for states 3 and 2. Its kind of confusing still if the unpublish message could point to either.

File: README.md [hunk 2]
```diff
@@ -36,6 +36,18 @@
...
 
+## Running a game server 
+
+- Import you game.json file into /taro/src/ directory
+
+To run the game server, execute the following command:
+```
+npm run server
+```
+
```
Comment (sloont) [inline README.md:47]: Why are we adding this when we still have the ability to fetch game.json from api?

File: src/client.js [hunk 3]
```diff
@@ -2,6 +2,9 @@ showMiniMap = false;
 showAllLayers = false;
 curLayerPainting = 'floor';
 let mouseIsDown = false;
+		
+
+
```
Comment (sloont) [inline src/client.js:7]: maybe just one newline, otherwise our eslint will get mad.

File: src/client.js [hunk 4]
```diff
@@ -139,22 +142,18 @@ const Client = IgeClass.extend({
 			this.isActiveTab = !document.hidden;
 		});
 
+		// create a global variable and use it in client JS
+
+
```
Comment (sloont) [inline src/client.js:147]: same thing with multiple newlines

File: src/client.js [hunk 5]
```diff
@@ -139,22 +142,18 @@ const Client = IgeClass.extend({
...
-					url: `${this.host}/api/game-client/${gameId}`,
-					dataType: 'json',
-					type: 'GET',
-					success: (game) => {
-						//
-						resolve(game);
-					}
-				});
-			} else {
+			if (gameId && window.location.hostname !== 'localhost') {
```
Comment (sloont) [inline src/client.js:153]: how can we test loading with ejs from localhost with BE/GS?

File: src/client.js [hunk 6]
```diff
@@ -139,22 +142,18 @@ const Client = IgeClass.extend({
...
+				return resolve({status: 'success', data: gameDetails.gameData});
+			}
+			else  {
```
Comment (sloont) [inline src/client.js:156]: else should be on same line as close and open bracket

**Review by sloont — CHANGES_REQUESTED** [review]

File: engine/components/network/net.io/net.io-client/index.js (hunk 1 above)
Comment (CLIcoder) [reply engine/components/network/net.io/net.io-client/index.js:298]: We don't need to be specific on the technical side of messages, what the user needs to know is that the game is not published on the current server anymore / connection lost

Comment (CLIcoder) [reply engine/components/network/net.io/net.io-client/index.js:298]: Case 1 is for guest users

Comment (CLIcoder) [reply engine/components/network/net.io/net.io-client/index.js:298]: Case 2 is for logged In users

**Comment by pankaj9296** [comment]
Closing this PR since the gameJson loading flow has been updated and added in the following PR
https://github.com/moddio/taro/pull/376

**Pull request closed without merging**